base/system/winlogon/sas.c

```diff
@@ -880,7 +843,8 @@ HandleLogoff(
 
     SwitchDesktop(Session->WinlogonDesktop);
 
-    // TODO: Play logoff sound!
+    /* Play logoff sound */
```

Comment:
Comments like these do not add value, the function name is already descriptive enough.

Reply:
Removed.